iidm-network-api/src/main/java/eu/itesla_project/iidm/network/CurrentLimits.java

```diff
@@ -70,6 +70,13 @@
     float getPermanentLimit();
 
     /**
+     * Set the permanent limit in A.
+     * @param permanenLimit the permanent limit in A
+     * @return itself for method chaining
+     */
+    CurrentLimits setPermanenLimit(float permanenLimit);
```

Comment:
A "T" is missing in the name of the method and the parameter. Adapt the Javadoc accordingly.